src/store/plugins/sdk.js

```diff
@@ -61,7 +61,11 @@ export default (store) => {
                   txBase64: params.tx,
                   opt: {
                     modal: !permission,
-                    host: originUrl.host,
+                    app: {
+                      name: originUrl.host,
```

Comment:
```suggestion
                      name: originUrl.hostname,
```

Reply:
```
{
                    name: originUrl.host,
                    host: originUrl.host,
                    protocol: originUrl.protocol,
}
```

Is been computed twice in the same function. Consider moving it to a common variable app and shorthand in the  JSON usage.

Reply:
I like Subhod's suggestion. This might be useful when working with `IAppData`. For example:
```
function getAppDataFromUrl(url: string, name?: string): IAppData { ... }
```